### DISCUSSION TITLE
Support for SimpleFOC motor?

### DISCUSSION THREAD

**Discussion by ipsod** [zauberzeug/lizard#8] [Ideas]
Hi.

I'd like to use SimpleFOC motors.  I think that SimpleFOC fits in well with your projects (by being simple, comprehensible, performant, free software), and the code is compatible with ESP32.

Is this something you would be willing to support?

Thanks for all the code.

**Comment by rodja** [comment]
Thanks for reaching out and pointing us to SimpleFOC motors. They look very interesting. Would you like to provide a Lizard module as pull request? Merging this in to main would be our pleasure.

**Comment by ipsod** [comment]
Ok, great, thanks!  I'll see what I can do.

**Comment by falkoschindler** [comment]
I'll convert this issue into a feature request, since it doesn't seem to be actively developed right now.
